### DISCUSSION TITLE
how to update for Stable Diffusion WebUI Docker

### DISCUSSION THREAD

**Discussion by horoabb** [AbdBarho/stable-diffusion-webui-docker#447] [Q&A]
I use git pull to stable-diffusion-webui-docker/ , use git log oneline say  (a68734c (HEAD -> master, origin/master, origin/HEAD))
this is new update, but i use webui make one styles.csv , reload its disappear.

is not update? or how to check stable-diffusion-webui-docker version.
and i see docker auto-sd:51,this maybe relevant, new id is 54?     

thanks

**Comment by AbdBarho** [comment]
to update you need to do
```
git pull
docker compose --profile auto up --build
```
it should work, if not, please create an issue.
make sure you are on the master branch
